selfdrive/controls/controlsd.py

```diff
@@ -856,6 +856,8 @@ def step(self):
     start_time = sec_since_boot()
     self.prof.checkpoint("Ratekeeper", ignore=True)
 
+    self.is_metric = Params().get_bool("IsMetric")
```

Comment:
Is it a bad idea to read a param at this frequency?

Reply:
should be fine, but this re-creates the Params object every time